tests/test_flask.py

```diff
@@ -61,7 +62,7 @@ def test_index(self):
         output = self.app.get('/')
         self.assertEqual(output.status_code, 200)
 
-        expected = """
+        expected = b"""
```

Comment:
An alternative to ``b'''`` is to replace ``output.data`` by ``output.get_data(as_text=True)``, don't know which you prefer though :)